hrui/js/src/civihr-popup/civihr-popup.js

```diff
@@ -42,10 +40,10 @@ Prevent Popups to overflow tables.
         .attr('id', $closestEntity.id)
         .addClass($($closestEntity).attr('class'));
 
-      $clone
-        .css('top', +buttonOffset.top + $button.outerHeight())
-        .css('left', +buttonOffset.left - $clone.width() - $button.outerWidth());
-
+      $clone.css({
+        left: +buttonOffset.left - ($clone.width() - $button.outerWidth()),
+        top: +buttonOffset.top + $button.outerHeight()
+      });
       $clone.show();
 
```

Comment:
No empty line needed. There are also other places in the code where there is an empty line between function calls, could you check please (like here https://github.com/civicrm/civihr/pull/2180/commits/a88eb67df7dd48cd34e94a4daaaf05ecba99a3b7#diff-5aa2eb87442af5119577aea46f98fadcR98)?